src/ontology/mondo-edit.obo

```diff
@@ -329354,10 +329355,9 @@ xref: GARD:16201 {source="OMIM:617142"}
 xref: OMIM:617142 {source="MONDO:equivalentTo"}
 xref: UMLS:C4310695 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0007119 {source="MONDO:Redundant", source="OMIM:617142", source="Orphanet:250923/btnt"} ! isolated aniridia
-intersection_of: MONDO:0007119 ! isolated aniridia
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/19016 ! TRIM44
```

Comment:
It isn't correct to remove this axiom. TRIM44 is reported in OMIM:617142 to be associated with this disease.